src/Codeception/Configuration.php

```diff
@@ -646,7 +646,7 @@ private static function prepareParams($settings)
             }
 
             // .env and ini files
-            if (preg_match('~(\.ini|\.env)$~', $paramStorage)) {
+            if (preg_match('~(\.ini$|\.env)~', $paramStorage)) {
```

Comment:
that sounds complicated indeed.
Perhaps @DavertMik or @Naktibalda know more about this?